### PR TITLE
ROX-19494: Bump wait time and provide context for initialization.

### PR DESCRIPTION
**Pull request by porridge**
## Description

Hopefully this will help us catch the culprit that causes timeouts described in the ticket.
I also added:
- a few info statements for a good measure
- timestamps to the e2e script messages

## Checklist (Definition of Done)

- [x] ~Unit and integration tests added~
- [x] ~Added test description under `Test manual`~
- [x] ~Documentation added if necessary (i.e. changes to dev setup, test execution, ...)~
- [x] CI and all relevant tests are passing
- [x] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [x] ~Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.~
- [x] ~Add secret to app-interface Vault or Secrets Manager if necessary~
- [x] ~RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)~
- [x] ~Check AWS limits are reasonable for changes provisioning new resources~

## Test manual

Ci should be sufficient.

### CODE REVIEW

File: fleetshard/pkg/runtime/runtime.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ type Runtime struct {
 }
 
 // NewRuntime creates a new runtime
-func NewRuntime(config *config.Config, k8sClient ctrlClient.Client) (*Runtime, error) {
+func NewRuntime(ctx context.Context, config *config.Config, k8sClient ctrlClient.Client) (*Runtime, error) {
```
Comment (kurlov) [inline fleetshard/pkg/runtime/runtime.go:63]: Should we also pass this context to the `isReconcilePaused`?

**Review by kurlov — COMMENTED** [review]
Looks great, just one question

File: fleetshard/pkg/runtime/runtime.go (hunk 1 above)
Comment (porridge) [reply fleetshard/pkg/runtime/runtime.go:63]: I can't see it, on which line is the call?

Comment (kurlov) [reply fleetshard/pkg/runtime/runtime.go:63]: It's on [176](https://github.com/stackrox/acs-fleet-manager/pull/1229/files/229aa2b7d52cf353cc193a53290dff7fbc33744b#diff-405fdfb027fcdcb359ab757ce7660f1511430e92eb03d82d1bdb05882d1d6a17R176)

Comment (porridge) [reply fleetshard/pkg/runtime/runtime.go:63]: I don't think so. The context passed to `NewRuntime` is for setup. The context passed to the `func` passed to the ticker on [136](https://github.com/stackrox/acs-fleet-manager/pull/1229/files/229aa2b7d52cf353cc193a53290dff7fbc33744b#diff-405fdfb027fcdcb359ab757ce7660f1511430e92eb03d82d1bdb05882d1d6a17R136) is a different per-tick one.

**Review by kurlov — APPROVED** [review]
Looks great!

File: scripts/lib/log.sh [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
+        printf "$(date --iso-8601=ns) $*"
         echo
     } >&2
     exit 1
 }
 
 log() {
     # shellcheck disable=SC2059
-    printf "$*"
+    printf "$(date --iso-8601=ns) $*"
```
Comment (vladbologa) [inline scripts/lib/log.sh:14]: This doesn't seem to work on mac os:

```
date: illegal option -- -
usage: date [-jnRu] [-I[date|hours|minutes|seconds]] [-f input_fmt]
            [-r filename|seconds] [-v[+|-]val[y|m|w|d|H|M|S]]
            [[[[mm]dd]HH]MM[[cc]yy][.SS] | new_date] [+output_fmt]
```

Comment (porridge) [reply scripts/lib/log.sh:14]: FTR, this was reverted for another reason in #1242 .
I have a fixed version up for review in #1244

**Pull request merged**